src/main/java/nl/tudelft/jpacman/ui/ScorePanel.java

```diff
@@ -35,11 +35,7 @@
 	public static final ScoreFormatter DEFAULT_SCORE_FORMATTER = 
 			// this lambda breaks cobertura 2.7 ...
 			// player) -> String.format("Score: %3d", player.getScore());
-			new ScoreFormatter() {
-				public String format(Player p) {
-					return String.format("Score: %3d", p.getScore());
-				}
-			};
+			p -> String.format("Score: %3d", p.getScore());
```

Comment:
This lambda causes cobertura to crash (as even indicated in the old comment).

The pom has now been configured to run with jacoco instead. So we should drop cobertura, and then merge this change.